modules/swagger-codegen/src/main/resources/php/api.mustache

```diff
@@ -358,7 +358,7 @@ use {{invokerPackage}}\ObjectSerializer;
         if (${{paramName}} !== null) {
             {{#isFile}}
             $multipart = true;
-            $formParams['file'] = \GuzzleHttp\Psr7\try_fopen(ObjectSerializer::toFormValue($file), 'rb');
+            $formParams['{{paramName}}'] = \GuzzleHttp\Psr7\try_fopen(ObjectSerializer::toFormValue(${{paramName}}), 'rb');
```

Comment:
@Alex-D thanks for the fix. I think `$formParams['{{paramName}}']` should be `$formParams['{{{baseName}}}']` as `baseName` store the original value of the parameter name (e.g. `date-time`)

Note: {{{ ... }}} will output the unescaped value

Please also run `./bin/php-petstore.sh` or `.\bin\php-petstore.bat` to update the Petstore samples so that the CI can verify the change.